### PR TITLE
Switch out 'gender gap' with gender neutral forms

### PR DESCRIPTION
**Pull request by theFeu**

### CODE REVIEW

**Comment by theFeu** [comment]
In German there are 2 different forms for nouns, that would correspond to "she/he", which i have switched out with a plural form, which would correspond to the usage of "they" in this case.

It improves the reading flow and improves inclusivity.

File: index-de.html [hunk 1]
```diff
@@ -76,7 +76,7 @@ <h2>Die längere Fassung</h2>
       Solltest du dich belästigt fühlen oder mitbekommen, dass jemand anders
       belästigt wird oder sonstige Probleme auftreten, wende dich bitte sofort
       an ein Mitglied des Veranstaltungspersonals. Diese sind an ihren
-      entsprechend markierten T-Shirts schnell zu erkennen.
+      entsprechend markierten T-Shirts und/oder Ausweisen schnell zu erkennen.
```
Comment (MattiSG) [inline index-de.html]: Is this still gender neutrality only? What is the aim of this change? 🙂

Comment (dnsmichi) [reply index-de.html]: It aims to make the sentence more clear, but agreed, not necessarily related to this PR. There are other rephrasing changes which make this more applicable to native German.

**Comment by theFeu** [comment]
> I support these changes. The German notation with 'he/she' needs to be replaced and made open for anyone, no matter which Gender they identify with. Some occasional word replacements help to modernize this document as well, like 'Fete' (afaik northern Germany) vs. 'Feier' (known in Germany, Austria).

I undid the 2 changes I made that are unrelated to the gender gap and will squash the commits.
I'll open another PR with exactly those two changes later :)

**Comment by ffrank** [comment]
FWIW, I'm more in favor of normalizing the gender gap in written language, rather than finding creative ways of avoiding the issue.
The German language just does not always offer neutral terminology, and the humanities have worked hard to give us actual means to cope. Let's use them.

**Comment by MattiSG** [comment]
And thanks for reviewing @dnsmichi @mbrochh! 😉

**Pull request merged**